wbooks-api/app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+
+      before_action :authenticate_user!
```

Comment:
We usually create an `ApiController` that is in charge or this. All the other controllers inheritates from this. You can keep it this way